test/libsolidity/smtCheckerTests/abi/abi_encode_call_simple_2.sol

```diff
@@ -0,0 +1,19 @@
+contract C {
+	function callMeMaybe(uint a, uint b) external {}
+
+	function abiEncodeSimple(uint x, uint y, uint z) public view {
+		require(x == y);
+		bytes memory b1 = abi.encodeCall(this.callMeMaybe, (x, z));
+		bytes memory b2 = abi.encodeCall(this.callMeMaybe, (y, z));
+		assert(b1.length == b2.length); // should hold but fails for now because function pointers like the above do not have a concrete value
```

Comment:
Are you planning on adding a constraint about the length?

Reply:
I use the lengths in these tests because I can't just do `assert(b1 == b2)`, so I'm kind of testing that b1 and b2 are the same by testing that there lengths are the same. We don't compute the lenghts precisely, so they can be only equal here for sure if the whole thing is the same.